internal/monitors/collectd/cpu/metadata.yaml

```diff
@@ -8,10 +8,48 @@ monitors:
     (the `cpu.utilization` metric).
 
     See https://collectd.org/wiki/index.php/Plugin:CPU
+  # There are no included metrics but shouldn't sendAll by default.
+  sendAll: false
```

Comment:
Why not send all?  It is a potential source of confusion to have a monitor that emits absolutely nothing when turned on.